package.json

```diff
@@ -3,9 +3,9 @@
   "name": "@zendeskgarden/react-components",
   "version": "0.0.0",
   "scripts": {
-    "build": "lerna run build --stream",
+    "build": "lerna run build --stream --concurrency 3",
     "build:analyze": "ANALYZE_BUNDLE=true yarn build:single",
-    "build:demo": "yarn build && lerna run build:demo --stream",
+    "build:demo": "lerna run build:demo --stream",
```

Comment:
We don't need to rebuild the packages as it's already accomplished during `yarn install`